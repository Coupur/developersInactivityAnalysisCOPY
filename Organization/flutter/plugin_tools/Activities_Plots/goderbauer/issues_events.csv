id,date,event,creator_login
1733518177,2018-07-13 21:48:41,review_requested,goderbauer
1733529572,2018-07-13 21:57:30,referenced,goderbauer
1733529574,2018-07-13 21:57:30,merged,goderbauer
1733529578,2018-07-13 21:57:30,closed,goderbauer
1733539246,2018-07-13 22:04:15,head_ref_deleted,goderbauer
1684297893,2018-06-15 21:26:58,mentioned,goderbauer
1684297894,2018-06-15 21:26:58,subscribed,goderbauer
1683738588,2018-06-15 15:58:50,referenced,goderbauer
1683738589,2018-06-15 15:58:50,merged,goderbauer
1683738595,2018-06-15 15:58:50,closed,goderbauer
1604071401,2018-05-01 20:13:27,referenced,goderbauer
1604071402,2018-05-01 20:13:27,merged,goderbauer
1604071405,2018-05-01 20:13:27,closed,goderbauer
1604071463,2018-05-01 20:13:30,head_ref_deleted,goderbauer
1571914259,2018-04-12 20:35:11,referenced,goderbauer
1571914261,2018-04-12 20:35:11,merged,goderbauer
1571914263,2018-04-12 20:35:12,closed,goderbauer
1459209104,2018-02-06 01:51:36,head_ref_force_pushed,goderbauer
1459219840,2018-02-06 02:02:58,closed,goderbauer
1176491545,2017-07-24 16:48:59,review_requested,goderbauer
1176494202,2017-07-24 16:50:41,review_requested,goderbauer
1177471495,2017-07-25 07:40:56,referenced,goderbauer
1177471496,2017-07-25 07:40:56,merged,goderbauer
1177471497,2017-07-25 07:40:56,closed,goderbauer
1177471607,2017-07-25 07:41:00,head_ref_deleted,goderbauer
1153305909,2017-07-06 18:54:49,review_requested,goderbauer
1154801310,2017-07-07 16:46:08,referenced,goderbauer
1154801312,2017-07-07 16:46:08,merged,goderbauer
1154801315,2017-07-07 16:46:08,closed,goderbauer
1154801396,2017-07-07 16:46:11,head_ref_deleted,goderbauer
1151358393,2017-07-05 17:35:56,review_requested,goderbauer
1151449592,2017-07-05 18:32:35,referenced,goderbauer
1151449593,2017-07-05 18:32:35,merged,goderbauer
1151449595,2017-07-05 18:32:35,closed,goderbauer
1151449667,2017-07-05 18:32:38,head_ref_deleted,goderbauer
1124139967,2017-06-14 22:03:22,referenced,goderbauer
1124139968,2017-06-14 22:03:22,merged,goderbauer
1124139969,2017-06-14 22:03:22,closed,goderbauer
1124140000,2017-06-14 22:03:24,head_ref_deleted,goderbauer
1124132085,2017-06-14 21:56:43,referenced,goderbauer
1124132086,2017-06-14 21:56:43,merged,goderbauer
1124132088,2017-06-14 21:56:43,closed,goderbauer
1116362754,2017-06-08 22:20:59,review_requested,goderbauer
1116365198,2017-06-08 22:23:01,referenced,goderbauer
1116365199,2017-06-08 22:23:01,merged,goderbauer
1116365200,2017-06-08 22:23:01,closed,goderbauer
1111063211,2017-06-06 04:04:04,mentioned,goderbauer
1111063212,2017-06-06 04:04:04,subscribed,goderbauer
1112212659,2017-06-06 17:21:38,mentioned,goderbauer
1112212661,2017-06-06 17:21:38,subscribed,goderbauer
1096109751,2017-05-24 18:24:11,review_requested,goderbauer
1096121559,2017-05-24 18:31:37,referenced,goderbauer
1096121564,2017-05-24 18:31:37,merged,goderbauer
1096121568,2017-05-24 18:31:37,closed,goderbauer
1096121683,2017-05-24 18:31:40,head_ref_deleted,goderbauer
1080758492,2017-05-12 22:44:21,head_ref_force_pushed,goderbauer
1080760638,2017-05-12 22:47:17,head_ref_force_pushed,goderbauer
1080763151,2017-05-12 22:50:19,review_requested,goderbauer
1080763205,2017-05-12 22:50:24,review_requested,goderbauer
1080765123,2017-05-12 22:53:17,head_ref_force_pushed,goderbauer
1080766292,2017-05-12 22:54:54,head_ref_force_pushed,goderbauer
1080777129,2017-05-12 23:09:07,head_ref_force_pushed,goderbauer
1080786544,2017-05-12 23:23:35,head_ref_force_pushed,goderbauer
1080787849,2017-05-12 23:25:48,head_ref_force_pushed,goderbauer
1082650085,2017-05-15 17:02:45,mentioned,goderbauer
1082650086,2017-05-15 17:02:45,subscribed,goderbauer
1082971374,2017-05-15 20:19:15,head_ref_force_pushed,goderbauer
1083180595,2017-05-15 22:53:44,referenced,goderbauer
1083180596,2017-05-15 22:53:44,merged,goderbauer
1083180597,2017-05-15 22:53:44,closed,goderbauer
1124132103,2017-06-14 21:56:44,referenced,goderbauer
