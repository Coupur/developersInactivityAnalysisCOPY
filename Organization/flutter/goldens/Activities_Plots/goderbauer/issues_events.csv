id,date,event,creator_login
2076320623,2019-01-16 02:57:33,referenced,goderbauer
2076320624,2019-01-16 02:57:33,merged,goderbauer
2076320626,2019-01-16 02:57:34,closed,goderbauer
2076320675,2019-01-16 02:57:36,head_ref_deleted,goderbauer
1947537156,2018-11-05 22:24:13,referenced,goderbauer
1947537158,2018-11-05 22:24:13,merged,goderbauer
1947537160,2018-11-05 22:24:13,closed,goderbauer
1947537266,2018-11-05 22:24:16,head_ref_deleted,goderbauer
1840902369,2018-09-12 11:41:48,referenced,goderbauer
1840902371,2018-09-12 11:41:48,merged,goderbauer
1840902372,2018-09-12 11:41:48,closed,goderbauer
1840902474,2018-09-12 11:41:51,head_ref_deleted,goderbauer
1840855717,2018-09-12 11:17:57,referenced,goderbauer
1840855719,2018-09-12 11:17:57,merged,goderbauer
1840855732,2018-09-12 11:17:57,closed,goderbauer
1840855816,2018-09-12 11:18:00,head_ref_deleted,goderbauer
1838062886,2018-09-11 09:38:47,head_ref_force_pushed,goderbauer
1838063412,2018-09-11 09:39:01,referenced,goderbauer
1838063413,2018-09-11 09:39:01,merged,goderbauer
1838063419,2018-09-11 09:39:01,closed,goderbauer
1838063522,2018-09-11 09:39:04,head_ref_deleted,goderbauer
1714993881,2018-07-03 20:54:43,review_requested,goderbauer
